crates/project/src/lib.rs

```diff
@@ -42,39 +35,37 @@ pub use compile::{
 mod config;
 pub use config::{PathStyle, ProjectPaths, ProjectPathsConfig, SolcConfig};
 
-pub mod remappings;
-
 mod filter;
 pub use filter::{
     FileFilter, FilteredSources, SourceCompilationKind, SparseOutputFilter, TestFileFilter,
 };
-use solang_parser::pt::SourceUnitPart;
 
 pub mod report;
 
-pub mod utils;
+/// Utilities for creating, mocking and testing of (temporary) projects
+#[cfg(feature = "project-util")]
+pub mod project_util;
 
-use crate::{
-    artifacts::{Source, SourceFile, Sources, StandardJsonCompilerInput},
-    cache::CompilerCache,
-    error::{SolcError, SolcIoError},
-    sources::{VersionedSourceFile, VersionedSourceFiles},
-};
-use artifacts::{contract::Contract, output_selection::OutputSelection, Settings, Severity};
+use cache::CompilerCache;
 use compile::output::contracts::VersionedContracts;
+use compilers::multi::MultiCompiler;
 use derivative::Derivative;
-use error::Result;
+use foundry_compilers_artifacts::{
+    output_selection::OutputSelection,
+    sources::{Source, Sources},
+    Contract, Settings, Severity, SourceFile, StandardJsonCompilerInput,
+};
```

Comment:
we likely want to re-export 

foundry_compilers_artifacts as artifacts

so that you don't need to depend on all crates if you're already pulling in compilers